src/main/java/gov/nasa/cumulus/metadata/aggregator/MetadataFilesToEcho.java

```diff
@@ -233,6 +230,25 @@ public void readCommonMetadataFile(String file, String s3Location) throws IOExce
 		}
 	}
 
+	private void buildGranuleReferences(JSONArray granule_files) {
```

Comment:
It's not clear to me why this is needed -- can you provide additional details about this? Alternatively feel free to reach out to me directly and we can discuss

Reply:
Is this function to fix PODAAC-5089?
related to this conversation with cumulus team? https://jpl.slack.com/archives/C76M787MZ/p1677107704530469
however, I would image the code would say if the file extension is .png then set type to browse in order to command postToCMR to place 'GET RELATED VISUALIZATION' in ummg.   Could you please explain in more details?   thanks!

Reply:
This is removed